File: src/main/java/ua/softgroup/matrix/server/supervisor/producer/json/WorkPeriod.java
package ua.softgroup.matrix.server.supervisor.producer.json;

import java.time.LocalTime;

/**
 * @author Oleksandr Tyshkovets <sg.olexander@gmail.com>
 */
public class WorkPeriod {

    private LocalTime start;
    private LocalTime end;

    public WorkPeriod() {
    }

    public WorkPeriod(LocalTime start, LocalTime end) {
        this.start = start;
        this.end = end;
    }

    public LocalTime getStart() {
        return start;
    }

    public void setStart(LocalTime start) {
        this.start = start;
    }

    public LocalTime getEnd() {
        return end;
    }

    public void setEnd(LocalTime end) {
        this.end = end;
    }

}

File: src/main/scala/ua/softgroup/matrix/server/supervisor/producer/json/WorkPeriod.scala
package ua.softgroup.matrix.server.supervisor.producer.json

import java.time.LocalTime

import scala.beans.BeanProperty

/**
  * @author Oleksandr Tyshkovets <olexandr.tyshkovets@gmail.com>
  */
class WorkPeriod {

  @BeanProperty
  var start: LocalTime = _

  @BeanProperty
  var end: LocalTime = _

  def this(start: LocalTime, end: LocalTime) {
    this()
    this.start = start
    this.end = end
  }

  override def toString = s"WorkPeriod(start=$start, end=$end)"

}
